### ISSUE TITLE
Beginners difficulty text does not meet color contrast standards

### ISSUE THREAD

**Issue by silvnikolovska** [silvnikolovska/demoblaze.com#1]
Element has insufficient color contrast of 3.63 (foreground color: #868688, background color: #ffffff, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1
